external-crates/move/move-compiler/src/expansion/ast.rs

```diff
@@ -412,7 +422,7 @@ pub enum FieldBindings {
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum LValue_ {
-    Var(ModuleAccess, Option<Vec<Type>>),
+    Var(Mutability, ModuleAccess, Option<Vec<Type>>),
```

Comment:
Same struct question

Reply:
Maybe... But Wouldn't the named fields just be something like `mut_: Mutability, var: ModuleAccess, type_: Option<Vec<Type>>`

This adds no additional clarity to me, but if it does for you, I will add it :)